modules/autofill_object_popup.py

```diff
@@ -0,0 +1,26 @@
+from selenium.webdriver.support import expected_conditions as EC
+
+from modules.page_base import BasePage
+
+
+class AutofillPopup(BasePage):
+    """
+    Page Object Model for the popups that have to do with autofill
+    """
+
+    URL_TEMPLATE = ""
```

Comment:
I previously had trouble with a blank `URL_TEMPLATE` but looks like it's working for you!